packages/jest-cli/src/TestRunner.js

```diff
@@ -115,6 +115,11 @@ class TestRunner {
     reporter.onRunStart && reporter.onRunStart(config, aggregatedResults);
 
     const onTestResult = (testPath, testResult) => {
+      if (testResult.testResults.length === 0) {
+        const EmptySuiteError = require('jest-util').EmptySuiteError;
+        throw new EmptySuiteError();
```

Comment:
Actually, since this is only used in one place/one package, can we keep this inside of `jest-cli` instead of `jest-util`?